Sources/Integration/Growler.m

```diff
@@ -49,17 +49,34 @@ - (void) growl:(Song*)song withImage:(NSData*)image isNew:(BOOL)n {
     NSUserNotification *not = [[NSUserNotification alloc] init];
     [not setTitle:title];
     [not setInformativeText:description];
-    [not setHasActionButton:true];
+    [not setHasActionButton:YES];
+    [not setActionButtonTitle: @"Skip"];
     
-    // Notification buttons
+    // Make skip button visible for banner notifications
+    // - see: https://github.com/indragiek/NSUserNotificationPrivate
     [not setValue:@YES forKey:@"_showsButtons"];
```

Comment:
So I think the 'setValue:forKey' selector is an NSObject method, so I think this will probably still work even if they deprecate this API.  I added a try/catch block that 'gracefully' handles this if the key doesn't exist.  Is that okay?